### DISCUSSION TITLE
'prec_acc_nc' and convective vs. non-convective rainfall in output files

### DISCUSSION THREAD

**Discussion by tpartrid** [hytest-org/hytest#411] [Dataset Questions]
Hi!

I've been processing conus404-hourly-onprem to force an offline version of WRF-Hydro. I'm wondering if the conus404 prec_acc_nc variable contains both convective and non-convective rainfall from WRF. It was my understanding that these are separated within WRF output files, but I don't see a corresponding prec_acc_c variable within the conus404 dataset.

Thanks for your help!

**Comment by amsnyder** [comment]
@AndreasPrein - do you have any insights on this question? Or @pnorton-usgs?

**Comment by pnorton-usgs** [comment]
@tpartrid The partitioning of convective versus grid-scale precipitation only occurs when using one of the cumulus parameterization schemes. The 4-km grid spacing of the CONUS404 model allowed us to do convection-permitting runs meaning we did not use any of the convective parameterization schemes and instead allowed model physics to explicitly resolve convection. As a result the model does not output convective precipitation. As a result the `PREC_ACC_NC` variable contains the total amount of precipitation (convective + grid-scale) and, in the raw model output, the convective precipitation variable is zero for the entire model run.

**Reply by tpartrid** [reply]
Ah, thanks @pnorton-usgs! I had it backwards in my mind for some reason - that partitioning was done when resolving convection... thanks for the clarification.

**Comment by jds485** [comment]
I think this is a related question: Does `PREC_ACC_NC` include rain and snow (and other) precipitation? Or only rain?

**Reply by amsnyder** [reply]
@aubreyd - do you know where someone could find this information? I (very briefly) tried to search the WRF manual for this variable to find more info and didn't find anything.

**Reply by aubreyd** [reply]
My guess is it is all precip forms, but tagging the master @lulinxue

**Reply by lulinxue** [reply]
@jds485 Yes, PREC_ACC_NC includes all precipitation types including rain, snow, and graupel.

**Reply by amsnyder** [reply]
Thanks @lulinxue !
